src/libraries/permissionUtils/usePermission.ts

```diff
@@ -0,0 +1,10 @@
+import { useSelector } from "react-redux";
+import { IState, TPermission } from "../../types";
+
+export const usePermission = (name: TPermission): boolean => {
+  const permissions = useSelector<IState, string[]>(
+    (state) => state.main.me?.data?.permission || []
+  );
+
+  return Boolean(permissions.find((permission: string) => permission === name));
```

Comment:
Why do we have to use a `Boolean` object here? Maybe a `!!` would do the job in a more performant fashion, wouldn't it?
```suggestion
  return !!permissions.find((permission: string) => permission === name);
```

Reply:
I don't know what is more performant, as long as there is a performance problem...